src/styles/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+  padding: 32px 16px 17px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    align-self: center;
+  }
+
+  &__title {
+    margin: 40px 0 4px;
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__id {
+    color: $secondary-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__reviews {
```

Comment:
Have you deployed the latest changes?

![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/5505764d-181f-43ee-96d5-f4b3f9c20d79)

You always should run `npm run deploy` after any changes. [Instruction](https://mate-academy.github.io/layout_task-guideline/#codenpm-run-deploycode-to-publish-to-github-pages), point 20